addons/fleet/i18n/fleet.pot

```diff
@@ -392,7 +397,6 @@ msgid "Closed"
 msgstr ""
 
 #. module: fleet
-#: model:ir.model.fields,field_description:fleet.field_fleet_vehicle__co2_standard
 #: model:ir.model.fields,field_description:fleet.field_fleet_vehicle_model__co2_standard
 msgid "Co2 Standard"
 msgstr ""
```

Comment:
This file doesn't need to be changed, right?

Reply:
> This file doesn't need to be changed, right?

Yes is necessary because our `O` is uppercase.   `Co2 Standard` ->  `CO2 Standard`  thank you